azurerm/internal/services/compute/virtual_machine_scale_set.go

```diff
@@ -1236,20 +1236,17 @@ func VirtualMachineScaleSetAutomatedOSUpgradePolicySchema() *schema.Schema {
 	return &schema.Schema{
 		Type:     schema.TypeList,
 		Optional: true,
-		ForceNew: true,
```

Comment:
This probably requires docs update as well, normally docs state that "Changing this forces a new resource to be created"?

Reply:
Yep, both windows and linux scaleset docs need an update.

Reply:
Docs updated

Reply:
@favoretti - is this OK now?

Reply:
@hbuckle LGTM, thank you, but a maintainer has to approve still.